### PR TITLE
CHE-1922 add paging for factory list

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Adds  paging for factory list for UD.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/1922

Signed-off-by: Oleksii Orel <oorel@codenvy.com>
selection_055

### CODE REVIEW

**Comment by benoitf** [comment]
ok for me
but it's quite hard to read as there are a lot of 'cleanup/style/rewording' changes not directly related to paging improvement

File: dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html [hunk 1]
```diff
@@ -118,15 +118,15 @@
...
       </ng-form>
     </che-label-container>
-    <!--Workspace RAM--Temporary not available //TODO add when implemented>
-   <!-- <che-label-container che-label-name="RAM">
+    <!--Workspace RAM-->
+    <che-label-container che-label-name="RAM">
       <ng-form name="factoryInformationForm">
         <che-workspace-ram-allocation-slider
-          ng-model="factoryInformationCtrl.copyOriginFactory.workspace.environments[0].machines[0].limits.ram"
+          ng-model="factoryInformationCtrl.copyOriginFactory.workspace.environments[0].machineConfigs[0].limits.ram"
```
Comment (ashumilova) [inline dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html]: no **machineConfigs** - **machines** instead

Comment (olexii4) [inline dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html]: done

File: dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html [hunk 2]
```diff
@@ -118,15 +118,15 @@
         </div>
       </ng-form>
     </che-label-container>
-    <!--Workspace RAM--Temporary not available //TODO add when implemented>
-   <!-- <che-label-container che-label-name="RAM">
```
Comment (ashumilova) [inline dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html]: if you uncomment this section, then provide the valid RAM value retrieving. Check "memoryLimitBytes" machine's attribute.

Comment (olexii4) [inline dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html]: done

**Comment by ashumilova** [comment]
+1

**Pull request merged**